src/Core/Impl/Extensions/StringExtensions.cs

```diff
@@ -302,5 +302,15 @@ public static bool TryGetNextNonEmptySpan(this string s, char separator, ref (in
             lineEnding = lineEnding ?? Environment.NewLine;
             return string.Join(lineEnding, s.SplitLines());
         }
+
+        public static int GetStableHash(this string s) {
```

Comment:
Why not use SHA256?

Reply:
Need an int for Id field required by BSON/LiteDb. SHA256 yields `byte[]`, no? Would need to xor then anyway.

Reply:
Ah, right. I thought it should be string.